### PR TITLE
Rest of the details

### PR DESCRIPTION
**Pull request by dorchard**
I think I have covered everything else here now about the reviewing process and the evaluation.  I also did various editing and cleaning up.

### CODE REVIEW

**Review by cassgvp — COMMENTED** [review]
For "available", could we make it explicit that an open-source license is required?

I've added some detail in the process. Still some bits to work out/publish e.g. minimal required technical experience of reviewers

**Comment by dorchard** [comment]
Great thanks @cassgvp! I added a readme file and put in about the open source license into the 'available' badge criteria. 

Re expectations:  "- Each review can be expected to take ==XXX== hours to complete, including direct communications with the submitting author via our review portal".  From my experience I would suggest this is probably a days work in total including all the communication and writing of review reports. This would likely be not in one block though. Does that sound reasonable?

File: badges.md
Comment (acocac) [inline badges.md:11]:
```suggestion
- [ ] An open-source license has been applied (for guidance, see https://choosealicense.com/).
```

File: badges.md
Comment (acocac) [inline badges.md:7]:
```suggestion
- [ ] Author-created artefacts relevant to the paper are publically accessible.
```

File: badges.md
Comment (acocac) [inline badges.md:8]:
```suggestion
- [ ] Relevant data sets are provided or are accessible where possible.
```

File: README.md
Comment (acocac) [inline README.md:6]: I recommend we mention previous work e.g. the Reproducibility Challenge hosted in CI2023. I'll replace the DOI with the position paper (when published). The paper proposes artefact evaluation as part of the vision of trust and reward in future CI conferences.

```suggestion
Climate Informatics, like many other communities and fields, has software at its heart. Underlying most publications is a novel piece of software playing some critical role, e.g., embodying a model, processing or analysing data, or producing a visualisation. In order for such software artefacts to have the most impact, they should be available, functional, and reusable, such that other researchers can benefit from the work, verify the claims of the paper, and then build upon the software to do more great work. These ideals are summarised by the FAIR principles of data, which can be applied to software: research software should be Findable, Accessible, Interoperable, and Reusable. In order to help promote [FAIR software](https://www.nature.com/articles/s41597-022-01710-x) and following the positive outcomes of the [2023 Reproducibility Challenge](https://doi.org/10.5281/zenodo.1037613), Climate Informatics is embarking, for the first time, on an _Artefact Evaluation phase_ for full paper submissions, after those submissions have been accepted for publication as the conference proceedings in [Environmental Data Science](https://www.cambridge.org/core/journals/environmental-data-science) following the traditional peer-review process. Artefact Evaluation provides an opportunity to embed the values of reproducibility into the publication process in a lightweight opt-in fashion, thus encouraging authors to make software available and the results of the paper reproducible.
```

File: README.md
Comment (acocac) [inline README.md:12]: Use a period for consistency with other bullet points.

```suggestion
* [process.md] Describes in detail the rationale, process, and evaluation criteria in place.
```

File: badges.md
Comment (acocac) [inline badges.md:29]: Add DOIs of FAIR data and FAIR4RS papers.

```suggestion
- [ ] Norms and standards of the research community for artefacts of this type are strictly adhered to (e.g., [FAIR data](https://doi.org/10.1038/sdata.2016.18) and [FAIR4RS](https://doi.org/10.1038/s41597-022-01710-x)).
```

File: process.md
Comment (acocac) [inline process.md:3]: Include the suggestion of the previous work in 2023 Reproducibility Challenge.

```suggestion
Climate Informatics, like many other communities and fields, has software at its heart. Underlying most publications is a novel piece of software playing some critical role, e.g., embodying a model, processing or analysing data, or producing a visualisation. In order for such software artefacts to have the most impact, they should be available, functional, and reusable, such that other researchers can benefit from the work, verify the claims of the paper, and then build upon the software to do more great work. These ideals are summarised by the FAIR principles of data, which can be applied to software: research software should be Findable, Accessible, Interoperable, and Reusable. In order to help promote [FAIR software](https://www.nature.com/articles/s41597-022-01710-x) and following the positive outcomes of the [2023 Reproducibility Challenge](https://doi.org/10.5281/zenodo.1037613), Climate Informatics is embarking, for the first time, on an _Artefact Evaluation phase_ for full paper submissions, after those submissions have been accented for publication as the conference proceedings in [Environmental Data Science](https://www.cambridge.org/core/journals/environmental-data-science) following the traditional peer-review process. Artefact Evaluation provides an opportunity to embed the values of reproducibility into the publication process in a lightweight opt-in fashion, thus encouraging authors to make software available and the results of the paper reproducible.
```

File: process.md
Comment (acocac) [inline process.md:32]:
```suggestion
- Each reviewer will be assigned 2 submissions.
```

File: process.md
Comment (acocac) [inline process.md:33]:
```suggestion
- 2-hour on-boarding session to get everyone up to speed.
```

File: process.md
Comment (acocac) [inline process.md:34]:
```suggestion
- Each review can be expected to take ==XXX== hours to complete, including direct communications with the submitting author via our review portal.
```

File: process.md
Comment (acocac) [inline process.md:35]:
```suggestion
- Final reviews should be submitted by ==XXX==.
```

File: process.md
Comment (acocac) [inline process.md:49]:
```suggestion
Authors will be encouraged to push improvements to the artefact whenever possible. For example, if essential documentation is missing, then rather than submitting a negative evaluation or providing only the reviewers with the required information, the authors should provide that information via a versioned/updated artefact.
```

File: process.md
Comment (acocac) [inline process.md:55]:
```suggestion
Artefact submissions require three parts, as described below. ==Authors will be invited to attend reviewer training, to receive full guidance on the expectations and processes for submission described below.==
```

File: process.md
Comment (acocac) [inline process.md:59]:
```suggestion
2. A non-institutional URL pointing to either: a single file containing the artefact (recommended), or the address of a public source control repository. The URL must be non-institutional to protect the anonymity of reviewers. Acceptable URLs can be obtained from Google Drive, Dropbox, Gitlab, Zenodo, and other providers. URLs may be generated using reviewer-only links. ==Do we prefer a versioned URL (e.g. DOI) here, or is that only expected if "accessible"==
```

Comment (acocac) [reply process.md:59]: I suggest we avoid recommending Google Drive and/or Dropbox. Instead, we should promote accepted version control platforms for software and data.

File: process.md
Comment (acocac) [inline process.md:61]:
```suggestion
3. A hash certifying the version of the artefact at submission time: either an md5 hash of the single file (use the md5 or md5sum command-line tool to generate the hash), or the full commit hash for the repository (e.g., from git reflog --no-abbrev).
```

File: process.md
Comment (acocac) [inline process.md:63]:
```suggestion
Artefacts do not need to be anonymous as reviewers will be aware of author identities.
```

File: process.md
Comment (acocac) [inline process.md:79]:
```suggestion
    - For unsupported claims, explain why they are omitted. For example, this can be reasonable if it requires very long runs on clusters, or involves data sets that are proprietary. In this case, the authors should provide some mechanism by which claims can be tested to establish the veracity of the approach, e.g., a smaller run or a different sample data set, with expected outputs.
```

File: process.md
Comment (acocac) [inline process.md:89]:
```suggestion
A good way to package artefacts is as a virtual machine (VM). VMs give an easily reproducible environment that is somewhat resistant to bit rot. They also give reviewers confidence that errors or other problems cannot cause harm to their machines. Source code artefacts should use Docker or another build tool to manage all compilation and dependencies. This improves the odds that the reviewers will be able to quickly and painlessly install the artifact — without getting lost in environment issues ([e.g. what Python do I need?!](https://xkcd.com/1987/)).
```

File: process.md
Comment (acocac) [inline process.md:93]:
```suggestion
There will be a 1-hour informal session for authors ==and reviewers== who are considering submitting artefacts to give an overview of the approach. This will happen online after CI2024 and informed via the dedicated [webpage for AE](https://alan-turing-institute.github.io/climate-informatics-2024/artefact-evaluation/) in the conference website.
```

**Review by acocac — CHANGES_REQUESTED** [review]
I've added some minor changes and other suggestions to consider in the PR.

It'd be great to add a glossary section if possible.

File: README.md
Comment (cassgvp) [reply README.md:6]: I think I'd like to put in a bit more structure back into the README, but will do this after this PR is merged :)

**Review by cassgvp — APPROVED** [review]
Very happy with this! Please feel free to merge now, @dorchard, if you'd like, or wait for further review from @rolyp ?

**Pull request merged**
